### ISSUE TITLE
Question about CER and val_CER

### ISSUE THREAD

**Issue by FrancescoGigli** [Calamari-OCR/calamari#211]
Hi I have a question about the result provided by calamari-train.
After my training the result i got is `Results of epoch   30: CER: 0.1284 - ctc_loss: 17.6702 - loss: 17.6702 - val_CER: 0.3662 - val_ctc_loss: 68.5998 - val_loss: 68.5998'
I cannot understand the difference beetwen CER and val_CER... Why a value is bigger than the other?
Thank you

**Comment by ChWick** [comment]
The Character Error Rate (CER) is computed on both the training and validation data sets. The network is usually overfitting (too good results) on the training data which is why a second data set is required to verify the actual accuracy to expect on unseen data (the validation list). Usually there is also a third (so called test) list to measure actual performance.
Due to overfitting the CER on the train list is way lower that on the val list, however the val_CER is the one you would expect if you apply your model on actual new data. Here (at least) about 37% of the characters will be wrong, on average.
